src/ansys/systemcoupling/core/adaptor/impl/injected_commands.py

```diff
@@ -61,7 +61,14 @@ def _wrap_add_participant(
                 "support this form of 'add_participant'. Minimum version is '24.1'."
             )
 
-        return part_mgr.add_participant(participant_session=session)
+        if not hasattr(session, "system_coupling"):
```

Comment:
In practice, this requires pyFluent release `ansys-fluent-core >= 0.17.0`

Reply:
and at least 24.1.0 Fluent (I did at a check for this in PyFluent)

Reply:
> and at least 24.1.0 Fluent (I did at a check for this in PyFluent)

Yeah, I think that's the right place for that sort of check.